spec/factories/service.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :service, :class => "Ovirt::Service" do
+    initialize_with { new(:server => "", :username => "", :password => "") }
```

Comment:
Why not just the normal...

```ruby
factory :service ... do
  server   ""
  username ""
  password ""
end
```

Maybe it doesn't work because it's not AR?